.github/workflows/windows-release.yml

```diff
@@ -215,9 +215,7 @@ jobs:
         path: _build/_CPack_Packages/win64/NSIS/NSISOutput.log
 
     - name: Upload NSIS installer
-      uses: actions/upload-release-asset@v1.0.2
-      with:
-        upload_url: ${{ steps.get_release.outputs.upload_url }}
-        asset_path: _build/${{env.NSIS_NAME}}
-        asset_name: ${{env.NSIS_NAME}}
-        asset_content_type: application/octet-stream
+        uses: softprops/action-gh-release@v0.1.15
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: _build/${{env.NSIS_NAME}}
```

Comment:
The zip upload (lines 188-194) still uses `actions/upload-release-asset@v1.0.2`. Please also update this.